modules/analysis-common/src/main/java/org/elasticsearch/analysis/common/CommonAnalysisPlugin.java

```diff
@@ -296,6 +303,22 @@ public Map<String, AnalysisProvider<TokenizerFactory>> getTokenizers() {
         tokenizers.put("thai", ThaiTokenizerFactory::new);
         tokenizers.put("ngram", NGramTokenizerFactory::new);
         tokenizers.put("edge_ngram", EdgeNGramTokenizerFactory::new);
+        tokenizers.put("edgeNGram", (IndexSettings indexSettings, Environment environment, String name, Settings settings) -> {
```

Comment:
yeah, this should just be removed :). I think this pr can be closed.

Reply:
Ok, thanks for reviewing my PR :)